main_test.go

```diff
@@ -107,9 +107,16 @@ func benchmarkGithub(owner string, repo string, build string, testbuild bool) fu
 			// If there's a vendor dir, we need to update the `modules.txt` in there to reflect the replacement.
 			tc.exec(b, "go", "mod", "vendor")
 		}
-		// traefik fails to build if we don't upgrade the version go.opentelemetry.io/otel/sdk/log
+		// traefik needs a few tweaks in order to build successfully
 		if repo == "traefik" {
+			// it fails to build if we don't upgrade the version go.opentelemetry.io/otel/sdk/log
 			tc.exec(b, "go", "get", "go.opentelemetry.io/otel/sdk/log@latest")
+
+			// it fails to build if ./webui/static does not exist, so just create an empty folder
+			if stat, err := os.Stat(filepath.Join(tc.dir, "webui")); err == nil && stat.IsDir() {
+				tc.exec(b, "mkdir", "-p", "./webui/static")
+				tc.exec(b, "touch", "./webui/static/index.html")
```

Comment:
Shelling out to create directories and files? Feels a little excessive?

Reply:
changed!